### ISSUE TITLE
Additional explanations

### ISSUE THREAD

**Issue by kosankem** [MHH-RCUG/scrnaseq_app#69]
At some parts I would miss some explanations.

- [ ]  Import data: Upload: This step took some time, but the window that the data are being processed showed up very late. It would be nice to have here a short sentence and not only in the help section. E.g.: Please choose the rds file you want to upload. The upload and processing can take some minutes. Once the processing is finished, you will be automatically forwarded to the next step.
- [ ]  Import data: Select genes: E.g.: Please select the genes that you would like to display.
- [ ]  Settings: Heatmap: I miss an explanation what data, counts, RNA, .... display. How would I decide as user what I have to choose? 
- [ ] Download: E.g. The files will be downloaded with the size and resolution that you have chosen in the general settings section. To adjust the format you wish to export the plots in, please go back to this section and re-run the analysis.

**Comment by kosankem** [comment]
I was missing the information that, in "Import data; Select through gene list", I need to press "select list" after I had selected one already by the pull down menu. When I was running without pressing that buttom, there was no warning that I had not select anything, but it was just processing forever (maybe with all genes?). 


Amendment: I was only that one time able to continue the processing without selecting any gene. I honestly do not know how I had managed it. I have now tried different combinations and orders but I was not able to reproduce the same bug.

**Comment by kosankem** [comment]
I have now figured out how I could create the error resulting in endless processing without selection of any gene.
It works when you perfome a first analysis and then you go back to "Input Data" and delete all selected genes. The button to continue is then gone from that section, but it is possible to go to "Settings" - "General" and use the button "Render plots" there.
